packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1650,12 +1657,35 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
    */
   private _onAutofillClick = (): void => {
     if (this.props.allowFreeform) {
-      this.focus(this.state.isOpen);
+      this.focus(this.state.isOpen || this._processingTouch);
     } else {
       this._onComboBoxClick();
     }
   }
 
+  private _onTouchStart = (ev: TouchEvent<HTMLInputElement>): void => {
+    if (this._comboBoxWrapper.value && !('onpointerdown' in this._comboBoxWrapper)) {
```

Comment:
@dzearing @joschect I had to add the onTouchStart event with an if check like this for split buttons, contextual menu split buttons, and combo boxes to support unit testing, because if I were to try and do something similar with addEventListener instead of adding this callback directly to a React component, the callback would never get called and it would not be unit testable.

Specifically, what I'm doing is: if onPointerDown is supported, than we don't do anything with onTouchStart and later in the componentDidMount, we'll add the pointerdown event, otherwise, we'll support onTouchStart and not add the pointerdown event.

This would be a temporary fix until React adds support for onPointerDown, but what are your thoughts on doing something like this? I was not able to think of a better solution. Thanks!

Reply:
Got it.

I added other comments. let me know when you've resolved.